pyscf/nao/gw_iter.py

```diff
@@ -327,22 +329,23 @@ def gw_chi0_mv(self,dvin, comega=1j*0.0, dnout=None):
             # padding m<n i.e. negative occupations' difference
             for n in range(vs+1,nf):
                 for m in range(n-vs):  
-                    nm2v_re[n,m],nm2v_im[n,m] = 0.0,0.0
+                    nm2v_re[n, m],nm2v_im[n, m] = 0.0,0.0
 
-        nb2v = self.gemm(1.0, nm2v_re, self.xvrt[s]) # real part
-        ab2v_re = self.gemm(1.0, self.xocc[s].T, nb2v, 1.0, ab2v_re)
+        nb2v = self.gemm(1.0, nm2v_re, self.xvrt[spin]) # real part
+        ab2v_re = self.gemm(1.0, self.xocc[spin], nb2v, 1.0, ab2v_re, trans_a=1)
```

Comment:
I don't see any accumulation for `ab2v_re`, then it should be

    ab2v_re += self.gemm(1.0, self.xocc[spin], nb2v, 1.0, ab2v_re, trans_a=1)

instead of

    ab2v_re = self.gemm(1.0, self.xocc[spin], nb2v, 1.0, ab2v_re, trans_a=1)

Is it correct @kovalp ?